nvtabular/ops/categorify.py

```diff
@@ -238,8 +228,10 @@ def __init__(
         # Only support two kinds of multi-column encoding
         if encode_type not in ("joint", "combo"):
             raise ValueError(f"encode_type={encode_type} not supported.")
+        if encode_type == "joint" and vocabs:
```

Comment:
Maybe something like 
```suggestion
        if encode_type == "joint" and vocabs is not None:
```

instead?

We're seeing unittest failures on this line with errors like ```E       ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all()```